### DISCUSSION TITLE
Unable to access SOC after AWS security onion 2 AMI installation

### DISCUSSION THREAD

**Discussion by GeonGod** [Security-Onion-Solutions/securityonion#11171] [Unsupported Versions]
I installed a secure onion2 AMI on AWS. The installation is complete, but I can't connect to SOC.
The current situation is that an Elastic IP is set on the onion instance, and the instance's public IP is entered in the "Enter a single IP address or IP range in CIDR notation to allow" part during installation. Any problem with this?
The installation is then complete, but the web interface is not accessible. Should I start separately? please tell me the solution
Please let me know if you have more documentation on installing the AWS secure onion2 AMI. :(

**Comment by dougburks** [comment]
> and the instance's public IP is entered in the "Enter a single IP address or IP range in CIDR notation to allow" part during installation. Any problem with this?

Instead of entering the instance's public IP address, have you tried entering the IP address or range of where your web browser will be connecting from?

> Please let me know if you have more documentation on installing the AWS secure onion2 AMI. :(

Assuming you're using the new Security Onion 2.4, have you reviewed the Amazon Cloud Image section of our 2.4 docs?
https://docs.securityonion.net/en/2.4/cloud-amazon.html

Also see:
https://docs.securityonion.net/en/2.4/configuration.html
https://docs.securityonion.net/en/2.4/post-installation.html
https://docs.securityonion.net/en/2.4/administration.html#administration
https://docs.securityonion.net/en/2.4/firewall.html
